lib/Errors.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 
 class UnhandledState extends Error {
-  constructor(transition, fromState) {
-    const message = `Transition from ${fromState} resulted in ${transition}`;
+  constructor(alexaEvent, transition, fromState) {
+    const message = `${alexaEvent.intent.name} went unhandled`;
```

Comment:
@armonge Could we add the previous state ? It could be very helpful if is not coming from `entry`